lib/ansiblelint/cli.py

```diff
@@ -36,6 +36,9 @@ def abspath(path, base_dir):
 
 
 def expand_to_normalized_paths(config, base_dir=None):
+    # config can be None (-c /dev/null)
+    if not config:
```

Comment:
If it can be `None`, why not check for it?

Reply:
Comparision with None would not be pythonic and would rely on implementation details. I could as well introduce a bug where and empty dictionary (or similar structure) would be considered "as having content".

Is it clear to everyone that this change does NOT introduce toleration of broken file paths? They will continue to be fatal errors. That is only about loading empty config files, where empty loose concept (no config items specified, it could still be a non empty file, like one with comments).